### DISCUSSION TITLE
Has dependency location changed in v0.1.14-pre?

### DISCUSSION THREAD

**Discussion by mkastner** [zigzap/zap#44] [Q&A] [answered]
In the README.md for version v0.1.14-pre the location for registering the zap dependency is "before exe.install()":

`Then, in your build.zig's build function, add the following before exe.install():`

While in the current release's README.md the location is "before b.installArtifact(exe)"

I'm just asking, because there is no exe.install in the build.zig generated by init-exe.

Or is there no exe.install, because I am on the latest zig nightly build?

**Comment by renerocksai** [comment, marked as answer]
I am out just about to present at a conference and will look it up later but I assume the installExe is one occurrence that fell through the cracks when updating the code and readme to zig 0.11. Always, the code is the source of truth if in doubt.
Zap is pinned to zig 0.11.0 (officially released version) btw. So ymmv with current master, esp. when it comes to updates of zig's standard library.
I will fix the readme later.

**Reply by mkastner** [reply]
@renerocksai It just caught my eye. By all means, I am grateful for your work and I am grateful for the features you've implemented in such a short period of time. Especially the code quality is great.

I am in fact learning a lot by reading the sources and the tests.

Thank you so much!
